src/tests/ftest/util/daos_utils.py

```diff
@@ -64,7 +64,22 @@ class DaosCommand(CommandWithSubCommand):
         # Sample get-attr output - no line break.
         # 04/19-21:16:32.66 wolf-3 Pool's attr2 attribute value:
         # 04/19-21:16:32.66 wolf-3 val2
-        "pool_get_attr": r"\b(\S+)$"
+        "pool_get_attr": r"\b(\S+)$",
+        "container_query":
+            r"Pool UUID:\s+([0-9a-f-]+)\n" +
+            r"Container UUID:\s+([0-9a-f-]+)\n" +
+            r"Number of snapshots:\s+(\d+)\n" +
+            r"Latest Persistent Snapshot:\s+(\d+)\n" +
+            r"Highest Aggregated Epoch:\s+(\d+)",
+        # Sample get-attr output - no line break.
+        # 04/20-17:47:07.86 wolf-3 Container's attr1 attribute value:  04
+        # /20-17:47:07.86 wolf-3 val1
+        "container_get_attr": r"\b(\S+)$",
```

Comment:
Can an attribute value have spaces in it?

Reply:
Yes. I modified the regex to handle it.